### PR TITLE
Homepage career

### PR DESCRIPTION
**Pull request by Parth097**

### CODE REVIEW

File: src/app/components/elements/CareersBanner.tsx [hunk 1]
```diff
@@ -19,50 +21,92 @@ const CsAtWorkDescription = () => {
...
+      <div className="career-link-column">
+        <Link className="career-link" to={linkTo}>
+          {linkText}
+        </Link>
+      </div>
+    </CardBody>
+  </Card>
+);
+
+CareerCard.propTypes = {
```
Comment (KrisBoxAND) [inline src/app/components/elements/CareersBanner.tsx:50]: Small fyi, PropTypes is deprecated and is mostly redundant in TS apps, so don't worry about spending time setting these up. Unless there's a specific functional need for them in this application?

File: src/app/components/elements/CareersBanner.tsx [hunk 2]
```diff
@@ -19,50 +21,92 @@ const CsAtWorkDescription = () => {
...
+  text: string | JSX.Element;
+  linkTo: string;
+  linkText: string;
+  children?: React.ReactNode;
+}
+
+const CareerCard = ({ imgSrc, imgAlt, title, text, linkTo, linkText, children }: CareerCardProps) => (
+  <Card className="career-card">
+    {imgSrc && <CardImg variant="top" src={imgSrc} alt={imgAlt} className="career-media-row-image" />}
+    <CardBody className="career-card-body">
```
Comment (KrisBoxAND) [inline src/app/components/elements/CareersBanner.tsx:37]: superb👏

File: src/app/components/elements/CareersBanner.tsx [hunk 3]
```diff
@@ -19,50 +21,92 @@ const CsAtWorkDescription = () => {
...
+                  width="100%"
+                  height="100%"
+                  src={`https://www.youtube-nocookie.com/embed/${videoId}?enablejsapi=1&fs=1&modestbranding=1`}
+                  allowFullScreen
+                />
+              </div>
+            </CareerCard>
+          </Col>
+        </Row>
+        <div className="career-comment">
```
Comment (KrisBoxAND) [inline src/app/components/elements/CareersBanner.tsx:100]: can we adjust the width / margisn on this bottom comment section? It's slightly off centre on mobile: 

image

File: src/scss/common/careers.scss [hunk 4]
```diff
@@ -1,82 +1,129 @@
...
+
+  &-media-row {
+    display: flex;
+    justify-content: space-between;
+    gap: 100px;
+    flex-wrap: wrap;
+  }
+
+  &-media-row > .col,
+  &-text-row > .col {
```
Comment (KrisBoxAND) [inline src/scss/common/careers.scss:44]: are all these styles necessary with the flexing on the column? ReactStrap should be doing the vast majority of this for us right?

File: src/scss/common/careers.scss [hunk 5]
```diff
@@ -1,82 +1,129 @@
...
-      padding: 1.5rem;
+
+  &-link-column {
+    text-align: center;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  &-link {
```
Comment (KrisBoxAND) [inline src/scss/common/careers.scss:102]: A good example of why SCSS is still needed as a pair for bootstrap :) 

I just ask you review all of the display and layout relating in this file please!

**Review by KrisBoxAND — CHANGES_REQUESTED** [review]
Superb changes, much more aligned and consistent. Just a few comments from me :D

File: src/scss/common/careers.scss (hunk 4 above)
Comment (Parth097) [reply src/scss/common/careers.scss]: You are right we don't need those, in fact removing them actually aligned everything a bit better

File: src/scss/common/careers.scss (hunk 5 above)
Comment (Parth097) [reply src/scss/common/careers.scss:64]: Removed unused styling from the file.

**Review by KrisBoxAND — APPROVED** [review]
superb mate!

**Pull request merged**